### PR TITLE
EXAMPLE - New Analytics Screen View Modifier

### LINKED ISSUE #197: IMPROVE - Analytics Screen View Modifier
## Describe the feature you want to improve
* Right now there are 8 views that have this modifier`.analyticsScreen(name: "\(Self.self)")` on them
* Create a new file called `FirebaseAnalytics+Extension` with a new view modifier that uses this modifier (put this file in the `Utilities` folder)
   * The new modifier should be called `analyticsView`
* Go through and use the newly created modifier on each view and remove the `import FirebaseAnalyticsSwift` on all the views, where necessary

The goal is that the new view modifier file has only a single import, and that we no longer need to import `FirebaseAnalyticsSwift` on all the views 

## Describe alternatives you've considered
* N/A

### PR DESCRIPTION
**Pull request by mikaelacaron-cwc**
# What it Does
* Closes #197
* Create new `ViewModifier` called `AnalyticsView`
* Use the `analyticsView` on the `DashboardView`

# How I Tested
* Run the app
* Go to the `DashboardView` tab
* See the log in the console

# Notes
* N/A

# Screenshot
* N/A

### CODE REVIEW

File: Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift [hunk 1]
```diff
@@ -63,7 +62,7 @@ struct DashboardView: View {
                 }
                 .listStyle(.inset)
             }
-            .analyticsScreen(name: "\(Self.self)")
+            .analyticsView("\(Self.self)")
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift:65]: Does this need to use string interpolation or can you pass `Self.self` directly?

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Great work!! I have several comments that need resolved please. If you have any questions, you can comment directly on that thread, otherwise click the resolve button once you've addressed my comment. Once you fix everything, please click the re-review button
Re-review button

File: Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift (hunk 1 above)
Comment (mikaelacaron-cwc) [reply Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift:65]: Why?

**Comment by mikaelacaron** [comment]
LIVESTREAMED on My YouTube channel
https://www.youtube.com/live/-rYYKXrCyro?si=r9D0ii3D1T7OLhlM

**Comment by mikaelacaron** [comment]
Closing, this was an example on my [livestream](https://youtube.com/live/-rYYKXrCyro)

**Pull request closed without merging**